NN_zodiak.js

```diff
@@ -0,0 +1,185 @@
+//pokazywanie zodiaku dla nerthusa
+try
+{
+
+nerthus.zodiac = {}
+nerthus.zodiac.calculate = function ()
+{
+    //zmienne do maszynki obliczającej
+    const date = new Date()
+    const day = date.getUTCDate()
+    const month = date.getUTCMonth() + 1
+
+    let zodiac = month - 1
+    switch (month) //zmiana znaków w innych miesiącach
+    {
+        case 1:
+            if (day < 20)
+                zodiac = 12
+            break;
+        case 2:
+            if (day < 18)
+                zodiac--
+            break;
+        case 3:
+        case 4:
+            if (day < 20)
+                zodiac--
+            break;
+        case 5:
+        case 6:
+        case 12:
+            if (day < 21)
+                zodiac--
+            break;
+        case 7:
+        case 11:
+            if (day < 22)
+                zodiac--
+            break;
+        case 8:
+        case 9:
+        case 10:
+            if (day < 23)
+                zodiac--
+            break;
+    }
+
+    return zodiac
+}
+
+nerthus.zodiac.spot = {}
```

Comment:
jakbyś przerobił grafikę tak, żeby wszystkie znaki zodiaku były w jednej kolumnie albo rzędzie to byś nie potrzebował takiej gimnastyki z liczeniem spotu i id